### PR TITLE
numba_issue3

### PR DESCRIPTION
**Pull request by sayaikegawa**
Hey @jstac , I attached a link of the lecture [parallelization](https://python.quantecon.org/parallelization.html) to clarify the `multithreading` in the lecture [numba](https://python.quantecon.org/numba.html).

### CODE REVIEW

**Comment by Harveyt47** [comment]
Yep, looks good to me.

**Comment by shlff** [comment]
Thanks @sayaikegawa and @jstac , it looks okay.

File: source/rst/numba.rst [hunk 1]
```diff
@@ -54,7 +54,7 @@ The key idea is to compile functions to native machine code instructions on the
 
 When it succeeds, the compiled code is extremely fast.
 
-Numba is specifically designed for numerical work and can also do other tricks such as multithreading.
+Numba is specifically designed for numerical work and can also do other tricks such as `multithreading <https://python.quantecon.org/parallelization.html>`_.
```
Comment (jstac) [inline source/rst/numba.rst]: Hi @sayaikegawa , could you please change the target link to 

<https://en.wikipedia.org/wiki/Multithreading_(computer_architecture)>

(It gives a better definition of multithreading.)  Thanks.

Comment (sayaikegawa) [reply source/rst/numba.rst]: Hi @jstac , I changed the target link.

**Comment by jstac** [comment]
Nice work, thanks @sayaikegawa.

(No need to review this one, @Harveyt47 and @shlff , I'll go ahead and merge.)

**Pull request merged**